dotcom-rendering/src/model/add-dots.ts

```diff
@@ -1,11 +1,4 @@
-// Current fix for incorrect use of Middot
-
-const transformDots = (html: string): string => {
-	return html.replace(
-		new RegExp('[•]', 'g'),
-		'<span data-dcr-style="bullet"></span>',
-	);
-};
+import { transformDots } from './transforDots';
```

Comment:
Actually, we use snake case here but we do use camel case in [quite a few other places](https://github.com/guardian/dotcom-rendering/tree/150fc2d81e6a66d9c3336185e874fc8cd0288546/dotcom-rendering/src/web/lib) 😐 

I completely agree this is better if consistent. I don't mind which pattern we use to be honest but I do think we need to keep the (pretty established industry convention) of naming _react components_ using `import { MyThing } from './MyThing.tsx'`

If someone wants to pick up this task to rename all the things then I guess they get to pick which way we go. But if we do do this work we probably also need to setup linting to enforce the result, otherwise we'll just end up with a mix again.